cypress/tests/ui/auth/change-password.spec.ts

```diff
@@ -0,0 +1,274 @@
+describe('Change password', () => {
+	const changePassword = {
+		title: 'Change Password',
+		oldPassword: 'Old Password',
+		newPassword: 'New Password',
+		confirmPassword: 'Confirm New Password',
+		btnSubmit: 'Update password',
+		alertTitle: 'Change password failed',
+	};
+	const user = {
+		username: Cypress.env('cognitoE2EUsername'),
+		password: Cypress.env('cognitoE2EPassword'),
+	};
+	const cognitoUrl = Cypress.env('cognitoEndpoint');
+	enum AUTH_VALIDATIONS {
+		CONFIRM_PASSWORD_REQUIRED = 'Confirm password is required',
+		CONFIRM_PASSWORD_NOT_MATCH = 'Passwords do not match',
+		NEW_PASSWORD_REQUIRED = 'New password is required',
+		OLD_PASSWORD_REQUIRED = 'Old password is required',
+		PASSWORD_INVALID = 'Your password must be at least',
+		PASSWORD_REQUIRED = 'Password is required',
```

Comment:
Mode this to other file.
Like `auth/exceptions/enum`
`auth/exceptions/constants`